src/lib/holocene/input/input.svelte

```diff
@@ -178,7 +178,7 @@
   }
 
   .input-container {
-    @apply surface-primary relative box-border inline-flex h-10 w-full items-center rounded border border-primary text-sm text-primary dark:focus-within:surface-primary focus-within:border-interactive focus-within:shadow-focus focus-within:shadow-primary/50 focus-within:outline-none dark:bg-transparent;
+    @apply surface-primary relative box-border inline-flex h-10 w-full items-center rounded border border-primary text-sm dark:focus-within:surface-primary focus-within:border-interactive focus-within:shadow-focus focus-within:shadow-primary/50 focus-within:outline-none dark:bg-transparent;
```

Comment:
Looks like we'll need to audit the filter search (specifically the border and background) since it uses various inputs

| dark | light |
|--|--|
|<img width="454" alt="Screenshot 2024-04-02 at 1 00 37 PM" src="https://github.com/temporalio/ui/assets/15069288/d064368e-5f22-4d36-ac1d-288f7c781326">|<img width="621" alt="Screenshot 2024-04-02 at 1 02 19 PM" src="https://github.com/temporalio/ui/assets/15069288/4ac9a862-1585-4f21-a9d0-5f148c1c4d26">|

Reply:
Thank for catching this Laura, I just made together a ticket for this :)